tests/conftest.py

```diff
@@ -190,10 +104,9 @@ def fixture_config():
 
         [core.layers]
         canopy_layers = 10
-        soil_layers = [-0.25, -1.0]
+        soil_layers = [-0.5, -1.0]
         above_canopy_height_offset = 2.0
```

Comment:
That looks right, as long as it does not break any other tests.

Reply:
It doesn't seem to and I think it's shared across your models.